app/controllers/admin/categories_controller.rb

```diff
@@ -133,4 +134,8 @@ def _categories
   def _category(id = params[:id])
     LinkedData::Client::HTTP.get(CATEGORIES_URL+ "/#{id.split('/').last}", { include: ATTRIBUTE_TO_INCLUDE })
   end
+
+  def categories_for_select
```

Comment:
this need to be a helper called, categories_selector_input that can be used everywhere.